spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/integration/inbound/GcsStreamingMessageSource.java

```diff
@@ -44,6 +44,14 @@ public GcsStreamingMessageSource(RemoteFileTemplate<BlobInfo> template) {
 		this(template, null);
 	}
 
+	/**
+	 * Creates a {@link GcsStreamingMessageSource} with a {@code comparator} which controls the order
+	 * that files are processed in.
+	 * @param template template making remote file calls to Google Cloud Storage
+	 * @param comparator defines the order that files should be processed based on {@link BlobInfo}.
+   *
+	 * @since 1.3
```

Comment:
```suggestion
	 * @since 1.2
```
Let's put it into the 1.2 branch as well.

Reply:
Will do.